### DISCUSSION TITLE
planning CT to CBCT

### DISCUSSION THREAD

**Discussion by Tibetan-eagle** [SuperElastix/elastix#1256] [General]
Hi! 

Does anyone have experience registering a planning CT to CBCT and worked around the registration parameters? I tried some online registration parameters - works well for prostate organ but not too well for bladder. 

Thanks!

TK

**Comment by stefanklein** [comment]
Thanks for your question. We don't have a ready parameter file, but since bladder can show large deformations, it might help to play with the settings of b-spline grid spacing and the multi-resolution strategy.

**Reply by Tibetan-eagle** [reply]
Thanks, I will give it a try, although I started off with the multi-resolution in the parameter file in the Elastix manual. 

Alternatively, I was using the Bladder mask of the moving (i.e. CT) to focus the registration to Bladder, but I am getting this error: 

AdvancedMattesMutualInformationMetric(0x55b6528cdd30): Too many samples map outside moving image buffer: 0 / 10000

Again this was using the parameter example from the Elastix manual. How can I fix this? Attached
[parameter.txt](https://github.com/user-attachments/files/17543179/parameter.txt)
 is the parameter for your reference
